packages/playground/src/components/Playground.tsx

```diff
@@ -39,7 +40,7 @@ export default function Playground({ themes, validators }: PlaygroundProps) {
     experimental_defaultFormStateBehavior: { arrayMinItems: 'populate', emptyObjectFields: 'populateAllDefaults' },
   });
   const [FormComponent, setFormComponent] = useState<ComponentType<FormProps>>(withTheme({}));
-  const [templates, setTemplates] = useState<Partial<TemplatesType>>();
+  const [otherFormProps, setOtherFormProps] = useState<Partial<FormProps>>({});
```

Comment:
Let's just capture all of the form props and spread them. `templates` can be included, so they don't need their own state variable.